src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -512,18 +504,26 @@ def add_entity(self, entity_key: str, entity_type: EntityType, entity_row: BigQu
                                            if isinstance(content, str)
                                            else content)
 
-    metadata_columns: ClassVar[Set[str]] = {
-        'version',
-        'JSON_EXTRACT_SCALAR(content, "$.schema_type") AS schema_type',
-        'BYTE_LENGTH(content) AS content_size',
-        'content'
-    }
-
-    data_columns: ClassVar[Set[str]] = metadata_columns | {
-        'descriptor',
-        'JSON_EXTRACT_SCALAR(content, "$.file_core.file_name") AS file_name',
-        'file_id'
-    }
+    @classmethod
+    @lru_cache
+    def columns(cls, entity_type: EntityType) -> Sequence[str]:
```

Comment:
@hannes-ucsc: "I understand why this was done but I still think it should be moved to the Plugin because column names are referenced all over the place and moving it to the Plugin eliminates the need to make this a class method."